src/main/java/com/org/jenkins/custom/jenkins/distribution/service/PackagerController.java

```diff
@@ -19,16 +21,22 @@
 import org.springframework.web.bind.annotation.RestController;
 import org.yaml.snakeyaml.Yaml;
 
-import static com.org.jenkins.custom.jenkins.distribution.service.generators.PackageConfigGenerator.*;
 @RestController
 @CrossOrigin
 @RequestMapping("/package")
 public class PackagerController {
 
     private static Util util = new Util();
-
+    private transient final PackageConfigGenerator packagerConfGen;
+    private transient final PackagerDownloadService packagerDownServ;
     private final static Logger LOGGER = Logger.getLogger(PackagerController.class.getName());
 
+    @Autowired
+    public PackagerController(final PackageConfigGenerator packageConfGen, final PackagerDownloadService packagerDownServ) {
```

Comment:
Needed to autowire this in order for it to be mocked so that it gets dependency gets injected during testing.